src/builtin-addons/core/script-definition-provider.ts

```diff
@@ -113,6 +118,10 @@ export default class CoreScriptDefinitionProvider {
       guessedPaths.push(pathLocation);
     });
 
+    this.resolvers.resolveTestScopeImport(root, importPath).forEach((pathLocation: string) => {
```

Comment:
this is not true, because addonName != addonPath, if import path contain addon name, we should resolve it's root or I missing something? could we check it like:

```js
if (importPath.startsWith(Project.name + '/tests/') || Project.addonsMeta.some(({name})=>importPath.startsWith(name+'/tests')) {
   resolveTestScopeImport..
} else {..old logic}

```

Reply:
this is not for addon names. If it is for addon name, then `addonImportPaths` should cover it. This is for cases where we are trying to search anything on the app level tests